eqs/api/api.toml

```diff
@@ -68,3 +68,8 @@ ERROR_hash = "A valid commitment hash is required. Commitment begin with CMTMNT_
 [route.healthcheck]
 PATH = [ "healthcheck" ]
 DOC = "Responds with JSON {\"status\": \"available\"}."
+
+[route.get_contract_address]
+PATH = [ "get_address/:assetdef" ]
```

Comment:
IIRC the route name has to be the same as the first segment of the path. Let's make both of these names more descriptive, like `get_wrapped_erc20_address`